playSong.py

```diff
@@ -188,12 +188,24 @@ def parseInfo():
 
 	return notes
 
+def adjustTempoForCurrentNote():
+    global playback_speed, infoTuple, storedIndex
+    tempo_changes = infoTuple[3]
```

Comment:
Are you sure `infoTuple` has enough information here? It's causing an `IndexError`.

See this file for testing, it has lots of tempo changes: [The Legend of Zelda, Ocarina of Time - Horse Race Theme.zip](https://github.com/Stereo101/python-MidiToVirtualPianoMacro/files/14549542/The.Legend.of.Zelda.Ocarina.of.Time.-.Horse.Race.Theme.zip)

Reply:
I gave wrong version